### ISSUE TITLE
Fatal Error : cannot redeclare __( )

### ISSUE THREAD

**Issue by benjamw** [phpmyadmin/phpmyadmin#12103]
Fatal error: Cannot redeclare __() (previously declared in phpmyadmin\libraries\sql-parser\src\common.php:17) in phpmyadmin\libraries\php-gettext\gettext.inc on line 290

The first instance checks for function exists, so maybe just switch the load order for common.php and the gettext module. Or remove the common.php file as it does nothing else.

### Server configuration
**Operating system**: Windows 8.1

**Web server:** Apache 2.4

**Database:** MySQL

**PHP version:** 7.03

**phpMyAdmin version:** 4.6.0.0 RC2

### Client configuration
**Browser:** Chrome

**Operating system:** Windows 8.1

**Comment by benjamw** [comment]
More info:

For some reason, not exactly sure why, the `session_start( )` call in `/libraries/session.inc.php` on line `101` is causing the `/libraries/sql-parser/src/Parser.php` file to be included, which includes the `libraries/sql-parser/src/common.php` file with the first instance of `__( )` in it.

That whole path is started through the `require './libraries/session.inc.php';` call in `/libraries/common.inc.php` on line `266`

The `gettext` version of the function is included via a `require_once GETTEXT_INC;` call in `/libraries/common.inc.php` on line `477`

**Comment by benjamw** [comment]
After thinking about it for a bit, it seemed odd that the call to `session_start( )` was loading classes that didn't seem to relate to sessions at all. Then I realized that some objects must be being stored in session (which is not the best idea in the first place). So I cleared my cookies and it seems to be working OK now.

Instead of storing entire objects in the session, it's much better to just store the data that will enable those objects to be rebuilt after session data has been loaded. But this is just my opinion. But I've been bitten by this issue in my own code before, and have learned from my mistakes.
